src/types.ts

```diff
@@ -113,13 +127,20 @@ export interface V4RFQFirmQuote {
     signedOrder: V4SignedRfqOrder;
 }
 
+export interface OtcOrderFirmQuoteResponse {
+    order?: OtcOrder;
```

Comment:
what's going on with these maybe types?

Reply:
The signature is a maybe because we'll want to reuse this for RFQT

The order shouldn't be a maybe

Reply:
Oh actually, the order is a maybe because apparently we allow MMs to return no response on this endpoint for our original `/price` and `/quote` endpoints